### PR TITLE
Feature/120455 nav updates

### PR DESCRIPTION
**Pull request by geoffgraham**
🔗 **Main site link** — Adds a link back to TheEventsCalendar.com from the main menu (.navbar) and is assigned as the first item in the nav.

🍯 **Sticky sidebar** — Knowledgebase posts now include a sticky sidebar, allowing users to search more content and access links to related posts from the current scroll position rather than having to go back to the top of the page.

🦶 **Updated footer** — Tested new markup for the footer and updated the styles accordingly.

🎨**Minor styling** — Did a quick audit of work and refined styles in a few places.

### CODE REVIEW

File: js/includes/navbar.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// Adds a link back to TEC.com in the main menu.
+addMainSiteLink( `
+	<li><a href="https://theeventscalendar.com">Main Site</li>
```
Comment (barryhughes) [inline js/includes/navbar.js]: Missing the closing `</a>` here.

File: scss/_pages/_page-article.scss [hunk 2]
```diff
@@ -6,6 +6,19 @@
 	padding: 2em;
 }
 
+.article-page .col-md-3 {
+	display: none;
+
+	@media (min-width: 1025px) {
+	background-color: $gray-20;
```
Comment (barryhughes) [inline scss/_pages/_page-article.scss]: Lines 13-18 should be indented one more tab.

File: js/includes/navbar.js (hunk 1 above)
Comment (geoffgraham) [reply js/includes/navbar.js]: 👍

File: scss/_pages/_page-article.scss (hunk 2 above)
Comment (geoffgraham) [reply scss/_pages/_page-article.scss]: 👍

**Pull request merged**